src/schema/contentPreference.ts

```diff
@@ -304,6 +337,28 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
     ): Promise<GQLEmptyResponse> => {
       await unfollowEntity({ ctx, id, entity });
 
+      return {
+        _: true,
+      };
+    },
+    block: async (
```

Comment:
Not 100% sure, but do we also need to introduce the other side backward compatibility?
So the current block flow to set contentPreference?

(Maybe you did this in other PR already, can't recall)

Reply:
I did add it, `blockKeyword` also saves to `Feedtag` with `blocked: true` and `blockSource` saves to `FeedSource`. 

Do we do anything additional with blocking?